documentation/how-to/standalone-datasets.en.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Standalone datasets
+---
+
+# Standalone datasets
+
+QField is able to directly open vector data and raster datasets without the need for a QGIS project. 
+
+## Supported standalone dataset formats
+
+The supported vector formats are:
+- Geopackage datasets (.gpkg);
+- GeoJSON (.geojson, .json);
+- KML (.kml, .kmz);
+- Shapefile (.shp);
+- GPS Exchange (.gpx);
+- Geography Markup Language (.gml);
+- MapInfo (.mif);
+- Spatialite (.db, .sqlite); and
+- FlatGeoBuf (.fgb)
+
+The supported raster formats are:
+- GeoTIFF (.tif, .tiff);
+- Georeferenced PDF / GeoPDF (.pdf);
+- JPEG2000 (.jp2);
+- JPEG (.jpg);
+- PNG (.png); and
+- WebP (.webp)
+
+!!! QField can handle several standalone datasets compressed into one ZIP archive; in this scenario, each dataset will be added as individual layers in QField.
+
+## Opening an vector or raster dataset
+
+:material-tablet-android:{ .device-icon } Fieldwork
+
+Please read the [storage access documentation](/get-started/storage/) to learn more on how standalone datasets are opened on your specific device.
+
+## Using a project as "base map" for standalone datasets
+
+:material-tablet-android:{ .device-icon } Fieldwork
+
+By default, the datasets will be overlying an OpenStreetMap XYZ layer. It is however possible to customize that by selecting a project stored onto the device QField is running that will act as a base map.
```

Comment:
```suggestion
By default, the datasets will be overlaying an OpenStreetMap XYZ layer. It is however possible to customize that by selecting a project stored onto the device QField is running that will act as a base map.
```

Reply:
@suricactus , that's an unfortunate typo right there :) :)